secret_store/src/key_server_cluster/math.rs

```diff
@@ -418,10 +429,108 @@ pub fn verify_signature(public: &Public, signature: &(Secret, Secret), message_h
 	Ok(combined_hash == signature.0)
 }
 
+
+/// Compute R part of ECDSA signature.
+#[cfg(test)]
+pub fn compute_ecdsa_r(nonce_public: &Public) -> Result<Secret, Error> {
```

Comment:
I think this is better explained as `x_coordinate(point)` than as "r", which requires diving a bit more into implementations of ecdsa

Reply:
I've added `public_x` and `public_y` (and used `public_x` in this function), but left `compute_ecdsa_r` - it is easier to think of `R` and `S` parts in ECDSA session (next PR). I.e. leave the fact that R is actually a `point.x` behind 'math-wall'.